src/utils/updates.ts

```diff
@@ -0,0 +1,112 @@
+import { E, ERef } from '@endo/eventual-send';
+import { makeAsyncIterableFromNotifier as iterateNotifier } from '@agoric/notifier';
+import {
+  iterateLatest,
+  makeLeader,
+  makeFollower,
+  Leader,
+} from '@agoric/casting';
+import { dappConfig } from 'config';
+import { identityMarshal } from 'utils/boardIdUnserializer';
+import type { Metrics, GovernedParams, BrandInfo, Brand } from 'store/store';
+
+const watchGovernance = async (
+  leader: Leader,
+  unserializer: any,
```

Comment:
nonblocking: for each watcher,
```suggestion
  marshall: IdentityMarshal,
```

Reply:
Actually these others are *not* using `IdentityMarshal`. They use the shared marshal from the wallet bridge so that brand the brands are the same as those from the purses etc. We just use the identity one to get the instance board id.

Added appropriate types to these though.